src/coreclr/jit/morph.cpp

```diff
@@ -14185,6 +14186,39 @@ GenTree* Compiler::fgMorphMultiOp(GenTreeMultiOp* multiOp)
                 }
                 break;
             }
+#endif
+
+            case NI_Vector128_Create:
```

Comment:
We should likely add back the handling (or do it in a separate PR) to have the same for any intrinsic that is `HWIntrinsicInfo::lookupCategory(intrinsicId) == HW_Category_IMM` (x86/x64) or `HWIntrinsicInfo::HasImmediateOperand(intrinsicId)` (ARM64)

Reply:
In those cases, the instruction has one or more non-SIMD operands (typically a `byte imm`) that is functionally "required" to be a constant, as otherwise it will risk expanding to a very expensive jump table.

Reply:
@tannergooding could you please look one more time? I've addressed your feedback and fixed regressions - they were coming from CreateScalarUnsafe, e.g. 
```csharp
public float Test()
{
    return Vector128.CreateScalarUnsafe(5.0f).ToScalar()
           + Vector128.CreateScalarUnsafe(5.0f).ToScalar();
}
```